beets/util/__init__.py

```diff
@@ -545,6 +546,16 @@ def hardlink(path, dest, replace=False):
                                   traceback.format_exc())
 
 
+def tmp_file_for(path=''):
+    """Returns a path to a named temporary file with the same file extension as
```

Comment:
```suggestion
    """Return a path to a named temporary file with the same file extension as
```

[PEP 257](https://www.python.org/dev/peps/pep-0257/) recommends imperative verbs for all docstrings.